jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/TextField.java

```diff
@@ -22,96 +19,157 @@
  **/
```

Comment:
Please see examples of writing JavaDoc for class descriptions.

Reply:
Done